drivers/d3d12/rendering_device_driver_d3d12.h

```diff
@@ -458,6 +461,9 @@ class RenderingDeviceDriverD3D12 : public RenderingDeviceDriver {
 	// Leveraging knowledge of actual usage and D3D12 specifics (namely, command lists from the same allocator
 	// can't be freely begun and ended), an allocator per list works better.
 	struct CommandBufferInfo {
+		// Store a self list reference to be used by the command pool.
+		SelfList<CommandBufferInfo> command_buffer_info_elem;
```

Comment:
I think we can do this so you don't need to add a constructor:

```suggestion
		SelfList<CommandBufferInfo> command_buffer_info_elem(this);
```

I'd swear we are already using this pattern somewhere else. Not a big deal, in any case.

Reply:
I'll absolutely make that change. I came across the following use of SelfList while digging through some examples of how to use it and figured that the standard might be to initialize it in a constructor:

https://github.com/godotengine/godot/blob/2b7ea6223bd0bdaa75706849c9dee1a82b2e2d01/core/object/worker_thread_pool.h#L74C1-L97C4

But I'm now realizing that constructor must only exist because of the bit fields.